src/DeviceStatusPage/IntrospectionCard.tsx

```diff
@@ -31,28 +32,40 @@ interface IntrospectionTableProps {
 const IntrospectionTable = ({
   deviceId,
   introspection,
-}: IntrospectionTableProps): React.ReactElement => (
-  <Table responsive>
-    <thead>
-      <tr>
-        <th>Name</th>
-        <th>Major</th>
-        <th>Minor</th>
-      </tr>
-    </thead>
-    <tbody>
-      {introspection.map((iface) => (
-        <tr key={iface.name}>
-          <td>
-            <Link to={`/devices/${deviceId}/interfaces/${iface.name}`}>{iface.name}</Link>
-          </td>
-          <td>{iface.major}</td>
-          <td>{iface.minor}</td>
+}: IntrospectionTableProps): React.ReactElement => {
+  const astarte = useAstarte();
+
+  return (
+    <Table responsive>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Major</th>
+          <th>Minor</th>
         </tr>
-      ))}
-    </tbody>
-  </Table>
-);
+      </thead>
+      <tbody>
+        {introspection.map((iface) => (
+          <tr key={iface.name}>
+            <td>
+              {astarte.token?.can(
+                'realmManagement',
+                'GET',
+                `/devices/${deviceId}/interfaces/${iface.name}/${iface.major}`,
```

Comment:
Here we want to verify access to device data (Appengine), not to the interface definition (Realm Management).

```suggestion
              {astarte.token?.can(
                'appEngine',
                'GET',
                `/devices/${deviceId}/interfaces/${iface.name}`,
```